docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -545,7 +545,8 @@ In such cases, use the `quarkus.oidc.token-state-manager.strategy` property to c
 
 If your chosen cookie strategy combines tokens and generates a large session cookie value that is greater than 4KB, some browsers might not be able to handle such cookie sizes.
 This can occur when the ID, access, and refresh tokens are JWT tokens and the selected strategy is `keep-all-tokens` or with ID and refresh tokens when the strategy is `id-refresh-token`.
-To workaround this issue,  you can set `quarkus.oidc.token-state-manager.split-tokens=true` to create a unique session token for each token.
+To workaround this issue, you can set `quarkus.oidc.token-state-manager.split-tokens=true` to create a unique session token for each token.
+Another way to workaround this issue is to store tokens in a database. For more information, please see the <<db-token-state-manager,Stateful token management>> section of this guide.
```

Comment:
```suggestion
Another way to workaround this issue is to store tokens in a database. For more information, please see the <<db-token-state-manager>> section of this guide.
```

Michelle explained it is no longer necessary to accompany such links with the text

Reply:
Interesting, thanks.

Reply:
Maybe you can drop it when pushing docs @sberyozkin, please? I don't want to trigger full CI just for dropping 3 words.

Reply:
Done.